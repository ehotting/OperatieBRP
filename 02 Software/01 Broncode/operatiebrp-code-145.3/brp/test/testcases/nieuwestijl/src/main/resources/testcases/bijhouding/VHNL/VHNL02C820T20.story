Meta:
@status                 Klaar
@regels                 R1587
@sleutelwoorden         voltrekkingHuwelijkInNederland
@usecase                UCS-BY.HG


Narrative:
BSN is < 9 posities

Scenario:   Personen Libby Thatcher (Ingeschrevene-Ingezetene, Niet NL Nat) en Piet Jansen (Onbekende) gaan trouwen, BSN is < 9 posities
            LT: VHNL02C820T20



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/VHNL02_reg_gesl_nm-Libby.xls

When voer een bijhouding uit VHNL02C820T20.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL02C820T20.txt

Then is in de database de persoon met bsn 690020041 niet als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 373230217 niet als PARTNER betrokken bij een HUWELIJK

Then lees persoon met anummer 8240349473 uit database en vergelijk met expected VHNL02C820T20-persoon1.xml











