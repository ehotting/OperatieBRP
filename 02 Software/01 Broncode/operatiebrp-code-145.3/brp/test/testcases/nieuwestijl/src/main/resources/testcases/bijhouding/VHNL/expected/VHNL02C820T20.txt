cvc-minLength-valid: Value '37323021' with length = '8' is not facet-valid with respect to minLength '9' for type 'Burgerservicenummer-s'.